plugins/woocommerce/src/Internal/Admin/Logging/LogHandlerFileV2.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Automattic\WooCommerce\Internal\Admin\Logging;
+
+use WC_Log_Handler_File;
+
+/**
+ * LogHandlerFileV2 class.
+ */
+class LogHandlerFileV2 extends WC_Log_Handler_File {}
```

Comment:
Two-part question:

- Should this in fact be the new default (instead of having merchants specify it via their `wp-config.php` file)?
- If not—or perhaps regardless of the answer to the above—and we expect the class to be referenced from locations outside of the plugin itself, it feels like placing this class inside the internal namespace isn't the right choice.

Reply:
> Should this in fact be the new default (instead of having merchants specify it via their wp-config.php file)?

Yes, I envision that this will become the new default, but not in this PR. I'd rather make the switch later when the logging improvements are more fully developed.

> we expect the class to be referenced from locations outside of the plugin itself, it feels like placing this class inside the internal namespace isn't the right choice.

🤔 That's a good point. I'm sort of on the fence about this. Right now this class is just a placeholder, but further on in this project I envision that it will be a completely separate implementation from `WC_Log_Handler_File` (so that the old file can eventually be deprecated). I don't know yet if the new handler would be something that we'd want people to be able to extend directly, or just use as a reference for writing their own handler. If we keep it in the internal namespace, we are more free to make changes to it without worrying about backcompat...

Reply:
> Yes, I envision that this will become the new default, but not in this PR.

Sounds good!

> If we keep it in the internal namespace, we are more free to make changes to it without worrying about backcompat.

That's definitely valid.